Tasks/ServiceFabricUpdateAppVersions/Update-PackageVersion.psm1

```diff
@@ -77,7 +77,7 @@
                     # Search the package files for differences (Unless the user doesn't want to log all changes and we already know the package xml changed)
                     $newPackagePath = Join-Path $NewPackageRoot $packageName
                     $oldPackagePath = Join-Path $OldPackageRoot $packageName
-                    if (Find-FileChanges $newPackagePath $oldPackagePath $LogIndent -LogAllChanges:$LogAllChanges)
+                    if ((Test-Path $newPackagePath) -and (Find-FileChanges $newPackagePath $oldPackagePath $LogIndent -LogAllChanges:$LogAllChanges))
```

Comment:
Adding another wrinkle, what if the 2nd build from above was not deployed to the production cluster (the package removal was a mistake found in integration testing)? Then if I tried to upgrade production from the package in 1 to the package in 3 it would not update the service (and I think would give an error)